numpyro/distributions/distribution.py

```diff
@@ -246,8 +271,25 @@ def expand(self, batch_shape):
         :return: an instance of `ExpandedDistribution`.
         :rtype: :class:`ExpandedDistribution`
         """
+        batch_shape = tuple(batch_shape)
```

Comment:
In Pyro, it is [common](http://pyro.ai/examples/tensor_shapes.html#It-is-always-safe-to-assume-dependence) to use `.expand([n])` so it is better to support it here.